srv/GraspPlanning.srv

```diff
@@ -0,0 +1,27 @@
+# Requests that grasp planning be performed for the target
+# returns a list of grasps to be tested and executed
+
+# the planning group used
+string group_name
+
+# the object to be grasped
+CollisionObject target
+
+# the name that the support surface (e.g. table) has in the collision map
+# can be left empty if no name is available
+string collision_support_surface_name
```

Comment:
Or for now just add a warning if someone tries to pass in more than one supporting surface. ROS messages are just much harder/annoying to change after the fact, so they should be general enough in the first run